### ISSUE TITLE
[Python] Inconsisteny in reflection

### ISSUE THREAD

**Issue by bergmannjg** [fable-compiler/Fable#2767]
### Description

Compiled field name is inconsistent with corresponding PropertyInfo
### Repro code

```f#
type Test = {
    firstName : string
    Age : int
}
```

### Expected and actual results

actual:

```py
def expr_238() -> TypeInfo:
    return record_type("Main.Test", [], Test, lambda: [["firstName", string_type], ["Age", int32_type]])


class Test(Record):
    def __init__(self, first_name: str, Age: int) -> None:
        super().__init__()
        self.first_name = first_name
        self.Age = Age or 0
    

Test_reflection = expr_238
```

expected:

```py
def expr_238() -> TypeInfo:
    return record_type("Main.Test", [], Test, lambda: [["firstName", string_type], ["Age", int32_type]])


class Test(Record):
    def __init__(self, firstName: str, Age: int) -> None:
        super().__init__()
        self.firstName = firstName
        self.Age = Age or 0
    

Test_reflection = expr_238
```

### Related information

* Fable version: fable-py 4.0.0-alpha-029
* Operating system Ubuntu 20.04

**Comment by dbrattli** [comment]
Thanks for reporting. This will be fixed in the next release

**Comment by dbrattli** [comment]
Note that the Python code will still be different from your expected, but consistent with PEP-8: [Method Names and Instance Variables](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables).

```py
def expr_0() -> TypeInfo:
    return record_type("Quicktest.Test", [], Test, lambda: [("first_name", string_type), ("Age", int32_type)])


class Test(Record):
    def __init__(self, first_name: str, Age: int) -> None:
        super().__init__()
        self.first_name = first_name
        self.Age = Age or 0


Test_reflection = expr_0
````

**Comment by alfonsogarciacaro** [comment]
Can this be closed @dbrattli? Unfortunately issues don't close automatically when merging to `beyond`.

**Comment by dbrattli** [comment]
Yes, this has now been fixed and released in 4.0.0-alpha-30
